src/pack/assemble127_extensions.ts

```diff
@@ -14,7 +14,14 @@ function assembleKeyValueString(self: ISO8583): DefaultError | Buffer {
   if (mtiCheck && validate && state) {
     const bitmaps_127 = self.assembleBitMap_127();
     const bmpsHex = self.getBitMapHex_127_ext();
-    let buff = Buffer.alloc(8, bmpsHex, 'hex');
+    const buff = Buffer.alloc(8, bmpsHex, 'hex');
```

Comment:
This variable is re-assigned, See the failing tests.

Reply:
Instantiate with just `let`

Reply:
I have made the changes, changed const to let

Reply:
@zemuldo would appreciate if you can review and approve if possible ASAP so that I can proceed with my deployment strategy :) Thanks